subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/BuildMetricsPlugin.kt

```diff
@@ -28,45 +28,50 @@ public open class BuildMetricsPlugin : Plugin<Project> {
             "Plugin must be applied to the root project but was applied to ${project.path}"
         }
 
-        val loggerFactory = GradleLoggerFactory.fromPlugin(this, project)
-        val logger = loggerFactory.create<BuildMetricsPlugin>()
-
         if (!project.pluginIsEnabled) {
-            logger.debug("Build metrics plugin is disabled")
+            project.logger.lifecycle("Build metrics plugin is disabled")
```

Comment:
The tradeoff - simple logging but we don't handle that log event by our handlers